### ISSUE TITLE
Inquiry about Data Sets for Natural Gas Compressibility Factor Modeling

### ISSUE THREAD

**Issue by NgBlaze** [aegis4048/GasCompressibiltiy-py#2]
Hello, @aegis4048 

I hope this message finds you well. I'm currently working on a machine learning project focused on natural gas compressibility factor modeling. Your repository on this topic has caught my attention, and I'm impressed by the work you've done.

I am reaching out to kindly inquire about the data sets you used for creating your model. I believe that having access to these data sets would greatly assist me in my project. I want to assure you that I take data confidentiality seriously, and any data shared with me will be kept strictly confidential and used solely for academic purposes.

If it's possible to share the data sets or provide any guidance on where I might find relevant data, I would greatly appreciate it. Your assistance would be invaluable in advancing my project.

Thank you for your time and consideration. Looking forward to your response.

Best regards

**Comment by aegis4048** [comment]
Hi @NgBlaze , this package is just an implementation of physical models presented in published scientific papers; there's no data I used. I just used the equations presented in those papers. The plots shown in the README.md are generated numbers from the gascompressiblity-py package.

Also if you are looking to implement a machine learning model to predict gas compressibility, I do not see a value in doing so because the physical models developed in the past can very accurately predict the z-factor...  unless you are doing this project for study purposes. The alternatives (physics models) already available on the internet (which includes this gascompressibility-py package and other z-factor libraries, such as [pyResToolbox)](https://github.com/mwburgoyne/pyResToolbox) can already predict the z-factor above 95% accuracy.
